.github/workflows/release-beta.yml

```
@@ -0,0 +1,51 @@
+name: Release Beta NPM Packages
+
+on:
+  push:
+    branches:
+      - feat_v3.x
+  workflow_dispatch:
+    tags:
```

actionlint (1.7.4)

expected "inputs" key for "workflow_dispatch" section but got "tags"

(syntax-check)